src/nvim/eval.h

```diff
@@ -68,6 +68,8 @@ enum {
     VV_LEN, /* number of v: vars */
 };
 
+#define MAX_FUNC_ARGS   20      /* maximum number of function arguments */
```

Comment:
I would suggest to convert this comment to a doxygen documentation string since you already have change to this line.